homeassistant/components/sonos/__init__.py

```diff
@@ -367,12 +367,7 @@ def async_discovered_player(
 
     async def setup_platforms_and_discovery(self):
```

Comment:
This function needs to be awaited to break the race

Reply:
109024fc3ce917c377615660c6f6ad61cb5c4af1